src/aiodynamo/client.py

```diff
@@ -907,19 +910,21 @@ async def _depaginate(
         task: Optional[asyncio.Task[Dict[str, Any]]] = asyncio.create_task(
             self.send_request(action=action, payload=payload)
         )
+        is_count = payload.get("Select") == "COUNT"
         try:
             while task:
-                result = await task
+                res = await task
```

Comment:
Out of curiosity, what's the rationale for `s/result/res/` ?

Reply:
Stops black from breaking up the line below when we use it